### PR TITLE
[VET-2029] Stardog.js changes for automapping service

### PR DESCRIPTION
**Pull request by SpiralP**
VET-2029

* requires develop version of stardog (probably future v8.1.0)

### CODE REVIEW

File: lib/dataSources.js [hunk 1]
```diff
@@ -167,6 +167,18 @@ const refreshMetadata = (conn, name, tableName = '') => {
   ).then(httpBody);
 };
 
+const suggestions = (conn, trigBody) => {
+  const headers = conn.headers();
+  headers.set('Content-Type', 'application/trig');
+  headers.set('Accept', 'application/ld+json');
```
Comment (anneeb) [inline lib/dataSources.js:173]: i would maybe make this configurable (`(conn, config, content)`)

File: package.json [hunk 2]
```diff
@@ -133,5 +133,5 @@
     "singleQuote": true,
     "trailingComma": "es5"
   },
-  "stardog-version": ">=7.0.0"
+  "stardog-version": ">=8.1.0"
```
Comment (anneeb) [inline package.json:136]: i think this is fine as `>=8.0.0`

File: lib/dataSources.js (hunk 1 above)
Comment (anneeb) [reply lib/dataSources.js:173]: also, can you update the docs with this?

Comment (SpiralP) [reply lib/dataSources.js:173]: configurable how? into fetch? why?

File: package.json (hunk 2 above)
Comment (SpiralP) [reply package.json:136]: you won't be able to use this new endpoint on 8.0.0 though; is that ok?

File: lib/dataSources.js (hunk 1 above)
Comment (anneeb) [reply lib/dataSources.js:173]: in case we want a return type that is not `ld+json`

File: package.json (hunk 2 above)
Comment (anneeb) [reply package.json:136]: yep that's ok as long as it's not breaking any existing features in 8+

File: lib/dataSources.js (hunk 1 above)
Comment (SpiralP) [reply lib/dataSources.js]: [`d2c4e12` (#276)](https://github.com/stardog-union/stardog.js/pull/276/commits/d2c4e12ba7bd3b9871d1076ce2ff4f6fd475cac7)

File: package.json (hunk 2 above)
Comment (SpiralP) [reply package.json]: [`c0086a5` (#276)](https://github.com/stardog-union/stardog.js/pull/276/commits/c0086a5175d6b4b91c04059cec00e1216bb2b0c7)

**Review by anneeb — APPROVED** [review]
LGTM

**Review by alex-r-bigelow — APPROVED** [review]

**Pull request merged**